crates/rpc/rpc-api/src/optimism.rs

```diff
@@ -55,6 +61,24 @@ pub struct OutputResponse {
     pub sync_status: SyncStatus,
 }
 
+/// https://github.com/ethereum-optimism/optimism/blob/c7ad0ebae5dca3bf8aa6f219367a95c15a15ae41/op-service/eth/types.go#L371
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub struct SystemConfig {
+    pub batcher_addr: Address,
+    pub overhead: B256,
+    pub scalar: B256,
+    pub gas_limit: u64,
```

Comment:
this must serialize with hex `alloy_rpc_types::serde::u64_hex`

Reply:
I have updated tests and 
utilize the result returned from a truly running node in tests, as provided
by QuickNode.  You can use 

```sh 
curl https://docs-demo.optimism.quiknode.pro/ \
 -X POST \
 -H "Content-Type: application/json" \
 --data '{"method":"optimism_syncStatus","params":[],"id":1,"jsonrpc":"2.0"}'
```
and 

```
curl https://docs-demo.optimism.quiknode.pro/ \
 -X POST \
 -H "Content-Type: application/json" \
 --data '{"method":"optimism_rollupConfig","params":[],"id":1,"jsonrpc":"2.0"}'
```

to get rpc result, in these response, the  "timestamp" is 1713235375, gasLimit is 30000000, both of them are not hex encoded. 

For more information, refer to the QuickNode documentation
at https://www.quicknode.com/docs/optimism/optimism_rollupConfig.